ygen/protogen.go

```diff
@@ -639,9 +649,23 @@ func safeProtoIdentifierName(name string) string {
 // between 1 and 2^29-1. The values 19,000-19,999 are excluded as these are explicitly
 // reserved for protobuf-internal use by https://developers.google.com/protocol-buffers/docs/proto3.
 func protoTagForEntry(e *yang.Entry) (uint32, error) {
-	// TODO(robjs): Replace this function with the final implementation
-	// once concluded.
-	return 1, nil
+	return fieldTag(e.Path())
```

Comment:
Above comment seems to be duplicated, these two should be consolidated.

Reply:
De-duplicated the comment. Sorry, this was a refactor for ensuring that the hashing code wasn't duplicated.